make/sandbox-ui.mk

```diff
@@ -1,235 +1,103 @@
-	git --git-dir=${RHDH_PLUGINS_TMP}/.git --work-tree=${RHDH_PLUGINS_TMP} remote add external ${AUTHOR_LINK}/rhdh-plugins.git
-	# fetch the branch
-	git --git-dir=${RHDH_PLUGINS_TMP}/.git --work-tree=${RHDH_PLUGINS_TMP} fetch external ${REMOTE_RHDH_PLUGINS_BRANCH}
-	# merge the branch with master
-	git --git-dir=${RHDH_PLUGINS_TMP}/.git --work-tree=${RHDH_PLUGINS_TMP} merge --ff-only FETCH_HEAD
-else
-	@echo "no pairing needed, using rhdh-plugins repo from master"
-	@$(MAKE) clone-rhdh-plugins 
-endif
-
-.PHONY: clone-rhdh-plugins
-clone-rhdh-plugins:
-	rm -rf ${RHDH_PLUGINS_TMP}; \
-	git clone --depth=1 https://github.com/redhat-developer/rhdh-plugins $(RHDH_PLUGINS_TMP) && \
-	echo "cloned to $(RHDH_PLUGINS_TMP)"
-
-.PHONY: push-sandbox-plugin
-push-sandbox-plugin:
-	$(MAKE) get-rhdh-plugins
-	cd $(RHDH_PLUGINS_TMP)/workspaces/sandbox && \
-	rm -rf plugins/sandbox/dist-dynamic && \
-	rm -rf red-hat-developer-hub-backstage-plugin-sandbox && \
-	yarn install && \
-	npx @janus-idp/cli@3.3.1 package package-dynamic-plugins \
-	  --tag $(IMAGE_TO_PUSH_IN_QUAY) \
-	  --platform $(PLATFORM) && \
-	podman push $(IMAGE_TO_PUSH_IN_QUAY)
-
-.PHONY: clean-sandbox-ui
-clean-sandbox-ui: HOST_NS=$(shell oc get projects -l app=host-operator --output=name -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | sort | tail -n 1)
-clean-sandbox-ui:
-	@oc delete ns ${SANDBOX_UI_NS}
-	@oc delete secret ${OPENID_SECRET_NAME} -n openshift-config
-	@oc delete usersignup ${SSO_USERNAME} -n ${HOST_NS}
-
-.PHONY: e2e-run-sandbox-ui
-e2e-run-sandbox-ui: HOST_NS=$(shell oc get projects -l app=host-operator --output=name -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | sort | tail -n 1)
-e2e-run-sandbox-ui: RHDH=https://rhdh-${SANDBOX_UI_NS}.$(shell oc get ingress.config.openshift.io/cluster -o jsonpath='{.spec.domain}')
-e2e-run-sandbox-ui:
-	@echo "Installing Playwright..."
+	@echo "Publishing and installing the Developer Sandbox Dashboard"
+	OPENID_SECRET_NAME=${OPENID_SECRET_NAME} DEVSANDBOX_DASHBOARD_NS=${DEVSANDBOX_DASHBOARD_NS}\
+		scripts/ci/manage-devsandbox-dashboard.sh -pp ${PUBLISH_UI} ${UI_REPO_PATH_PARAM} -ds ${DATE_SUFFIX} -qn ${QUAY_NAMESPACE} ${FORCED_TAG_PARAM} -du ${DEPLOY_UI}
+
+.PHONY: e2e-run-devsandbox-dashboard
+e2e-run-devsandbox-dashboard: HOST_NS=$(shell oc get projects -l app=host-operator --output=name -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | sort | tail -n 1)
+e2e-run-devsandbox-dashboard: RHDH=https://rhdh-${DEVSANDBOX_DASHBOARD_NS}.$(shell oc get ingress.config.openshift.io/cluster -o jsonpath='{.spec.domain}')
+e2e-run-devsandbox-dashboard:
 	$(eval PWGO_VER := $(shell grep -oE "playwright-go v\S+" go.mod | sed 's/playwright-go //g'))
 	@echo "Installing Playwright CLI version: $(PWGO_VER)"
 	go install github.com/playwright-community/playwright-go/cmd/playwright@$(PWGO_VER)
 	@echo "Installing Firefox browser for Playwright..."
 	$(GOPATH)/bin/playwright install firefox
```

Comment:
I know that it wasn't changed in this PR so take it as an suggestion for later improvement. 
There are two things:
* I think that it would be better/reliable to use GOBIN variable instead of GOPATH
* it could install playwright to a temp folder in the same way as we do it with ksctl: 
https://github.com/codeready-toolchain/toolchain-e2e/blob/14739a8bde3efb862a47be1fba1dd3b99659414d/make/ksctl.mk#L3
https://github.com/codeready-toolchain/toolchain-e2e/blob/14739a8bde3efb862a47be1fba1dd3b99659414d/make/ksctl.mk#L70